<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32,32,32);</string>
  </property>
  <widget class="QPushButton" name="guestButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>110</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-style: outset;
border-width: 2px;
    border-radius: 10px;
background-color: rgb(132,132,132);
font-size: 25px;
color: rgb(255,255,255);
}</string>
   </property>
   <property name="text">
    <string>  Guest  </string>
   </property>
  </widget>
  <widget class="QLabel" name="loginLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>221</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:rgb(255,255,255);
font-size: 30px;
}</string>
   </property>
   <property name="text">
    <string>Sign in</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:rgb(255,255,255);
font-size: 25px;
}</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>104</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-style: outset;
border-width: 2px;
border-radius: 10px;
font-size: 25px;
background-color: rgb(132,132,132);
color: rgb(255,255,255);
}</string>
   </property>
   <property name="text">
    <string>  Login  </string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgotPassButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>351</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-style: outset;
	border-width: 0px;
	color: red;
}</string>
   </property>
   <property name="text">
    <string>Forgot password?</string>
   </property>
  </widget>
  <widget class="QLabel" name="passLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:rgb(255,255,255);
font-size: 25px;
}</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userLineField">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>259</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passLineField">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>259</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>180</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	text-align: center;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
